app/controllers/katello/concerns/api/v2/repository_content_controller.rb

```diff
@@ -280,6 +272,25 @@ def check_repo_for_content_resource
             {content: resource_name, id: params[:id]}
         end
       end
+
+      def handle_cv_filter(collection, filter, filter_rule, params)
```

Comment:
Is it intentional that this method takes the filter as a param but refers to the `@filter` instance variable?

Reply:
This is still outstanding